src/lib/billing/billing.test.ts

```diff
@@ -0,0 +1,117 @@
+import nock from 'nock';
+import { test } from 'tap';
+
+import { config } from '../../app/app.test';
+
+import { BillingClient } from '.';
+
+// tslint:disable:max-line-length
+nock(config.billingAPI)
+  .get('/billable_events?range_start=2018-01-01&range_stop=2018-01-02&org_guid=org-guid-500-no-data').reply(500)
```

Comment:
Could these individual mocks be moved into the specific tests that they relate to so that it's easier to understand each test?